index.html

```diff
@@ -144,6 +144,116 @@ <h4 class="heading heading_level-4">Подушка</h4>
+            <div class="column-wrapper">
+                <div class="columns">
+                    <div class="column column_s-2 column_m-3 column_l-4">
+                        <div class="social">
+                            <a href="#" class="social__link">
+                                <img src="./styles/blocks/footer/images/social/facebook.svg" alt="facebook">
+                            </a>
+                            <a href="#" class="social__link">
+                                <img src="./styles/blocks/footer/images/social/twitter.svg" alt="twitter">
+                            </a>
+                            <a href="#" class="social__link">
+                                <img src="./styles/blocks/footer/images/social/vk.svg" alt="vk">
+                            </a>
+                            <a href="#" class="social__link">
+                                <img src="./styles/blocks/footer/images/social/instagram.svg" alt="instagram">
+                            </a>
+                        </div>
+                    </div>
+
+                    <div class="column column_s-2 column_m-3 column_l-4">
+                        <div class="stores">
+                            <a href="#" class="stores__link">
+                                <img src="./styles/blocks/footer/images/stores/apple.png" alt="instagram">
+                            </a>
+                            <a href="#" class="stores__link">
+                                <img src="./styles/blocks/footer/images/stores/google.png" alt="instagram">
+                            </a>
+                        </div>
+                    </div>
+
+                    <div class="column column_l-4">
+                        <div class="logo-monochrome">
+                            <a href="#" class="logo-monochrome__link">
+                                <img
+                                    src="./styles/blocks/footer/images/logo.svg"
+                                    alt="instagram"
+                                    class="logo-monochrome__image"
+                                >
+                            </a>
+                        </div>
+                    </div>
+
+                    <div class="column column_s-2 column_m-6 column_l-12">
+                        <div class="footer__line"></div>
+                    </div>
+
+                    <div class="column column_s-2 column_m-6 column_l-12">
+                        <div class="copyright">
+                            <p>
+                                © 2019 Группа компаний HeadHunter
+                            </p>
+                            <p>
+                                Сегодня на сайте 684 978 вакансий, 
+                                34 789 747 резюме, 858 923 компании 
+                                и за неделю 1 211 363 приглашения
+                            </p>
+                        </div>
+                    </div>
```

Comment:
не обязательно все заворачивать в колонки, если контент занимает всегда всю ширину
```suggestion
                <div class="footer__line"></div>

                <div class="copyright">
                    <p>
                        © 2019 Группа компаний HeadHunter
                    </p>
                    <p>
                        Сегодня на сайте 684 978 вакансий, 
                        34 789 747 резюме, 858 923 компании 
                        и за неделю 1 211 363 приглашения
                    </p>
                </div>
```

Reply:
вынес блоки за пределы сетки. ты прав, меньше кода с тем же результатом